### ISSUE TITLE
[ATOMIC CONTRACT]: Build out tech blog

### ISSUE THREAD

**Issue by Christopher-Stevers** [OpenQDev/OpenQ-Frontend#717]
### 📃 Description

Build out tech blog with different theme and font than regular blog. We are looking for some sort of cyber theme, so monospace fonts and electro colours - flourescent blues, greens -- and a dark background would be good. The tech blog should have its own category in netlify cms, but the content format should be the same as the current blog, except conditionally show the images dpending if their is a img src from netlify.

### 👨‍⚖️ Acceptance Criteria

Have design approved.
Pull request with a version that allows admins to create tech posts and displays them in a seperate list on the homepage.
Tech posts are comply with approved design.

### 💰 Budget

200 USDC

### 🖼 Screenshots + Mock Ups

No mock ups, please message me with your own design mock up, so I can approve it before you get into the code.

### 🎬 Where to Start

Basically copy the ./posts folder and related content that updates it.

### 📵 What's Out of Scope

Migrating my beyootiful blog to react.

### 🤳 User Story

1. When the  Admin goes to /admin/
2. They have the option to edit or create tech posts.
3. Users can access tech posts from homepage, without the homepage expanding below the fold on desktop.

### 🛠 Tools to Use

Use eleventy to manage the ssg of tech posts, just like regular posts.

### 🖇 Resources

Eleventy discord.
